modules/nw-ingress-route-secret-load-external-cert.adoc

```diff
@@ -6,36 +6,35 @@
 [id="nw-ingress-route-secret-load-external-cert_{context}"]
 = Creating a route with externally managed certificates
 
-You can configure {product-title} routes with third-party certificate management solutions by using the `.spec.tls.externalCertificate` field of the route API. You can reference externally managed TLS certificates via secrets, eliminating the need for manual certificate management. Using the externally managed certificate reduces errors ensuring a smoother rollout of certificate updates, enabling the OpenShift router to serve renewed certificates promptly. 
+[role="_abstract"] 
+You can configure {product-title} routes with third-party certificate management solutions by using the `.spec.tls.externalCertificate` field of the route API. You can reference externally managed TLS certificates via secrets, eliminating the need for manual certificate management.
 
-You can use externally managed certificates with both edge routes and re-encrypt routes.
+By using the externally managed certificate, you can reduce errors to ensure a smoother rollout of certificate updates and enable the OpenShift router to serve renewed certificates promptly. You can use externally managed certificates with both edge routes and re-encrypt routes.
 
 .Prerequisites
 
 * You must enable the `RouteExternalCertificate` feature gate.
```

Comment:
The feature gate `RouteExternalCertificate` is promoted from Tech Preview to GA since 4.19, so maybe we need additional info here since the changes are for 4.16+.